src/visualizers/VisualizerGrid.ts

```diff
@@ -0,0 +1,1067 @@
+        property11Visualization: getPropertyVisualization(
+            this.property11Visualization,
+            11
+        ),
+        property11MainColor: getPropertyColor(this.property11MainColor, 11),
+    }
+
+    checkParameters() {
+        const status = super.checkParameters()
+
+        return status
+    }
+    setup(): void {
+        this.sketch.createCanvas(400, 400)
+    }
+
+    draw(): void {
+        this.setPresets()
+
+        this.sketch.strokeWeight(0)
+
+        this.setOverridingSettings()
+
+        this.amountOfNumbers = Math.min(
+            this.amountOfNumbers,
+            this.seq.last - this.seq.first + 1
+        )
+
+        //Round up amount of numbers so that it is a square number.
+        const squareRootOfAmountOfNumbers = Number(
+            floorSqrt(this.amountOfNumbers)
+        )
+
+        this.amountOfNumbers =
+            squareRootOfAmountOfNumbers * squareRootOfAmountOfNumbers
+
+        //This is because 20 x 20 is 1:1 scaling.
+        const scalingFactor = 400 / squareRootOfAmountOfNumbers
+
+        this.setPathVariables(squareRootOfAmountOfNumbers, scalingFactor)
+
+        this.currentSequenceIndex = Math.max(
+            this.startingIndex,
+            this.seq.first - 1
+        )
+        let augmentForRowReset = 0n
+
+        for (
+            let iteration = 0;
+            iteration < this.amountOfNumbers;
+            iteration++
+        ) {
+            //Reset current sequence for row reset and augment by one.
+            if (this.currentDirection === Direction.StartNewRow) {
+                if (this.resetAndAugmentByOne) {
+                    this.currentSequenceIndex = Math.max(
+                        this.startingIndex,
+                        this.seq.first
+                    )
+                    augmentForRowReset++
+                }
+            }
+
+            this.currentSequenceIndex++
+
+            const currentNumber = this.getCurrentNumber(
+                this.currentSequenceIndex,
+                augmentForRowReset
+            )
+
+            this.fillGridCell(currentNumber, scalingFactor)
+
+            this.moveCoordinatesUsingPath(
+                squareRootOfAmountOfNumbers,
+                iteration,
+                scalingFactor
+            )
+        }
+        this.sketch.noLoop()
+    }
+
+    setPresets() {
+        if (this.preset != Preset.Custom) {
+            this.property0 = Property.None
+            this.property1 = Property.None
+            this.property2 = Property.None
+            this.property3 = Property.None
+            this.property4 = Property.None
+            this.property5 = Property.None
+            this.property6 = Property.None
+            this.property7 = Property.None
+            this.property8 = Property.None
+            this.property9 = Property.None
+            this.property10 = Property.None
+            this.property11 = Property.None
+        }
```

Comment:
Similarly, you should be doing setting a whole list of similar properties like this in a loop, not written out.